soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11613,7 +11613,7 @@ void func_8084BF1C(Player* this, GlobalContext* globalCtx) {
         phi_f2 = -1.0f;
     }
 
-    this->skelAnime.playSpeed = phi_f2 * phi_f0;
+    this->skelAnime.playSpeed = phi_f2 * phi_f0 + phi_f2 * CVar_GetS32("gClimbSpeed", 0);
```

Comment:
Is `gClimbSpeed` able to be 0? Would that result in a `playSpeed` of 0 here?

Reply:
Yes, `gClimbSpeed` actually defaults to 0, but since it's added to the original `playSpeed` value, that just means you get the default speed in that case.  When it's not zero, it always has the same sign as the default speed, so it will always increase positive speed and decrease negative speed, which means it always retreats from 0 and never approaches it.